guides/common/modules/ref_customizing-the-registration-templates.adoc

```diff
@@ -36,7 +36,7 @@ Snippet is used in the *Linux host_init_config default* template:
 
 * The `remote_execution_ssh_keys` snippet deploys SSH keys to the host only when the `host_registration_remote_execution` parameter is `true`.
 * The `insights` snippet downloads and installs the Red{nbsp}Hat Insights client when global parameter `host_registration_insights` is set to true.
-* The `puppetlabs_repo` and `puppet_setup` snippets downloads and installs puppet agent on the host (only when puppet master is present)
+* The `puppetlabs_repo` and `puppet_setup` snippets downloads and installs Puppet agent on the host (only when Puppet server is present)
```

Comment:
Perhaps it should say:
```suggestion
* The `puppetlabs_repo` and `puppet_setup` snippets downloads and installs Puppet agent on the host (only when a Puppet server is assigned)
```

My reasoning is that a Puppet server being present is ambiguous. Only when a host has a Puppetserver assigned, it'll be used. This means you can have hosts with and without Puppet in the same environment.

Reply:
Perfect, thanks. This felt unclear when I read it.

Reply:
Also, should it be plural?
```suggestion
* The `puppetlabs_repo` and `puppet_setup` snippets download and install Puppet agent on the host (only when a Puppet server is assigned)
```